doc/source/whatsnew/v0.15.0.rst

```diff
@@ -724,61 +722,61 @@ Other notable API changes:
   .. ipython:: python
      :okexcept:
 
-     s = Series(np.arange(3,dtype='int64'),
-                index=MultiIndex.from_product([['A'],['foo','bar','baz']],
-                                              names=['one','two'])
-               ).sort_index()
-     s
+     s = pd.Series(np.arange(3, dtype='int64'),
+                   index=pd.MultiIndex.from_product([['A'], ['foo', 'bar', 'baz']],
+                                                    names=['one', 'two'])
+                   ).sort_index()
+    
      try:
-        s.loc[['D']]
+         s.loc[['D']]
      except KeyError as e:
-        print("KeyError: " + str(e))
-
 - Assigning values to ``None`` now considers the dtype when choosing an 'empty' value (:issue:`7941`).
+         print("KeyError: " + str(e))
+
 
   Previously, assigning to ``None`` in numeric containers changed the
   dtype to object (or errored, depending on the call). It now uses
   ``NaN``:
 
-  .. ipython:: python
+  ..  ipython:: python
```

Comment:
there should be only space between the `..` and `ipython`. Here, and in all the cases that follow.

Reply:
Yes, i will work on it ASAP.